src/account_manager/operations/mod.rs

```diff
@@ -6,6 +6,8 @@ pub(crate) mod background_syncing;
 pub(crate) mod get_account;
 #[cfg(feature = "ledger_nano")]
 pub(crate) mod ledger_nano;
+#[cfg(all(feature = "participation", feature = "storage"))]
```

Comment:
It seems `participation` can't be used without `storage` so maybe make it so participation actually requires storage in the Cargo.toml file so you don't have to require both at the same time always, also make it less error prone for users